hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/client/OzoneQuota.java

```diff
@@ -136,10 +133,11 @@ public static OzoneQuota parseQuota(String quotaString)
       found = true;
     }
 
-    if (!found) {
-      throw new IllegalArgumentException(
-          "Quota unit not recognized. Supported values are BYTES, MB, GB and " +
-              "TB.");
+    if (Boolean.FALSE.equals(found)) {
```

Comment:
Since `found` is never `null`, can we change it to `boolean` instead?

Reply:
and also, might be because I have not caught up with the new hip code styles; I find !found to be far easier to read than Boolean.FALSE.equals(found)

Reply:
Thanks @adoroszlai and @anuengineer for the reviews.
Yeah ```found``` is never ```null```, I converted its type to primitive type, and the ```if-statement``` is more readable.
Fixed.